### PR TITLE
16 featureclient add auth pages

### LINKED ISSUE #16: feature(client): add auth pages

### PR DESCRIPTION
**Pull request by huilensolis**

### CODE REVIEW

File: apps/client/middleware.ts
Comment (Jes015) [inline apps/client/middleware.ts:1]: I have a question. Why are we using a middle-ware if we are going to use our own server?

File: apps/client/src/app/auth/sign-in/components/form/index.tsx
Comment (Jes015) [inline apps/client/src/app/auth/sign-in/components/form/index.tsx:1]: Hii. I wouldn't recommend to use a "index.tsx" file. It's better to call to the components by its name. That way you can identify the files easily.

File: apps/client/middleware.ts
Comment (huilensolis) [reply apps/client/middleware.ts:1]: The middleware runs for each request made to Next. Each time you make a get request to a page of next, it runs the middleware. so we can use the middleware to validate the user session, and if it has no valid session, we can deny access to specific paths of our routing on the client.

https://nextjs.org/docs/app/building-your-application/routing/middleware

File: apps/client/src/app/auth/sign-in/components/form/index.tsx
Comment (huilensolis) [reply apps/client/src/app/auth/sign-in/components/form/index.tsx:1]: okay, let me change it

**Review by Jes015 — APPROVED** [review]
All is ok. <3

**Pull request merged**